packages/@aws-cdk/aws-signer/package.json

```diff
@@ -79,20 +79,29 @@
     "pkglint": "0.0.0"
   },
   "dependencies": {
-    "@aws-cdk/core": "0.0.0"
+    "@aws-cdk/core": "0.0.0",
+    "constructs": "^3.2.0"
   },
   "peerDependencies": {
-    "@aws-cdk/core": "0.0.0"
+    "@aws-cdk/core": "0.0.0",
+    "constructs": "^3.2.0"
   },
   "engines": {
     "node": ">= 10.13.0 <13 || >=13.7.0"
   },
   "stability": "experimental",
-  "maturity": "cfn-only",
+  "maturity": "experimental",
   "awscdkio": {
     "announce": false
   },
   "publishConfig": {
     "tag": "latest"
+  },
+  "awslint": {
+    "exclude": [
+      "resource-attribute:@aws-cdk/aws-signer.SigningProfile.signingProfileProfileName",
+      "resource-attribute:@aws-cdk/aws-signer.SigningProfile.signingProfileProfileVersion",
+      "resource-attribute:@aws-cdk/aws-signer.SigningProfile.signingProfileProfileVersionArn"
+    ]
```

Comment:
Suppress lint error due to change attribute names. Is this correct?

Reply:
I'll take a look at why the linter is not correctly processing this, and revert back to you.

Reply:
Looks like I gave you the wrong direction. The values for `@attribute` need to be `ProfileName`, `ProfileVersion` and `ProfileVersionArn`. I've pushed a commit to this PR to fix this.

Apologies.